pennylane_qiskit/basic_aer.py

```diff
@@ -16,7 +16,7 @@
 evaluation and differentiation of Qiskit Terra's BasicAer simulator
```

Comment:
Might want to edit the docstring to better match use of `QiskitAer` now.

Reply:
I'd rather wait until we have a decision on how the Aer device will work. This PR makes the tests pass so we can continue development, it doesn't reflect a decision on how we should update the Aer device now that they are deprecating things.